<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesignRail</class>
 <widget class="QMainWindow" name="DesignRail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DesignRail</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="parametersGB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>261</width>
      <height>251</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Geometry Parameters</string>
    </property>
    <widget class="QLineEdit" name="HDlineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>26</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>00+(00/00); </string>
     </property>
     <property name="text">
      <string>1+(7/8)</string>
     </property>
    </widget>
    <widget class="QLabel" name="HDlabel">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>28</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>HD:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FDlineEdit">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>26</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00+(00/00); </string>
     </property>
     <property name="text">
      <string>3+(13/16)</string>
     </property>
    </widget>
    <widget class="QLabel" name="FDlabel">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>28</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>FD:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="BDlineEdit">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>64</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00+(00/00); </string>
     </property>
     <property name="text">
      <string>1+(1/8)</string>
     </property>
    </widget>
    <widget class="QLabel" name="BDlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>66</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BD:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="HWlineEdit">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>64</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00+(00/00); </string>
     </property>
     <property name="text">
      <string>2+(21/32)</string>
     </property>
    </widget>
    <widget class="QLabel" name="HWlabel">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>66</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>HW:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="WlineEdit">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>104</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00+(00/00); </string>
     </property>
     <property name="text">
      <string>0+(5/8)</string>
     </property>
    </widget>
    <widget class="QLabel" name="Wlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>106</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>W:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="BWlineEdit">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>100</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00+(00/00); </string>
     </property>
     <property name="text">
      <string>5+(1/2)</string>
     </property>
    </widget>
    <widget class="QLabel" name="BWlabel">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>105</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BW:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="R1HlineEdit">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>142</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00+(00/00); </string>
     </property>
     <property name="text">
      <string>0+(1/2)</string>
     </property>
    </widget>
    <widget class="QLabel" name="R1Hlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>144</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R1H:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="R2HlineEdit">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>142</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00+(00/00); </string>
     </property>
     <property name="text">
      <string>0+(1/16)</string>
     </property>
    </widget>
    <widget class="QLabel" name="R2Hlabel">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>144</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R2H:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RHBlineEdit">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>182</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00+(00/00); </string>
     </property>
     <property name="text">
      <string>0+(3/4)</string>
     </property>
    </widget>
    <widget class="QLabel" name="RHBlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>184</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RHB:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="alphalineEdit">
     <property name="geometry">
      <rect>
       <x>174</x>
       <y>182</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00˚00'; </string>
     </property>
     <property name="text">
      <string>14˚02'</string>
     </property>
    </widget>
    <widget class="QLabel" name="alphalabel">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>184</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>alpha:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="betalineEdit">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>220</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00˚00'; </string>
     </property>
     <property name="text">
      <string>1˚26'</string>
     </property>
    </widget>
    <widget class="QLabel" name="betalabel">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>222</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>beta:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="thetalineEdit">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>218</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string notr="true">00˚00'; </string>
     </property>
     <property name="text">
      <string>14˚02'</string>
     </property>
    </widget>
    <widget class="QLabel" name="thetalabel">
     <property name="geometry">
      <rect>
       <x>133</x>
       <y>220</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>theta:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="viewGB">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>5</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <widget class="GLDesign" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>231</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="drawPB">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>255</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>draw 
geometry</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawmesh">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>256</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>draw 
mesh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refineB">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>256</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>refine</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>256</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Workspace</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>545</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Rail"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSave_Rail">
   <property name="text">
    <string>Save Rail</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLDesign</class>
   <extends>QWidget</extends>
   <header location="global">gldesign.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
